test/test.js

```diff
@@ -774,3 +774,12 @@ test('fail-fast mode', function (t) {
 		t.end();
 	});
 });
+
+test('power-assert support', function (t) {
+	t.plan(2);
+
+	execCli('fixture/power-assert.js', function (err, stdout, stderr) {
+		t.ok(err);
+		t.true(stderr.indexOf('t.ok(a === \'bar\')\n') !== -1);
```

Comment:
Without power-assert, the output should be `false == true` because this assert is `t.ok()`.
I thought probably it renders `t.ok(a === 'bar')` only when power-assert works.

```js
test(t => {
	const a = 'foo';

	t.ok(a === 'bar');
	t.end();
});
```

Should I test this renders the diagram output somehow?

```
  t.ok(a === 'bar')
       |           
       "foo"       
```